app/Repositories/CurrikiGo/LmsSetting/LmsSettingRepositoryInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Repositories\CurrikiGo\LmsSetting;
+
+use App\Models\CurrikiGo\LmsSetting;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Collection;
+
+interface LmsSettingRepositoryInterface
+{
+
```

Comment:
Please define interface for additional functions like 'fetchAllByUserId' you defined in repository.
Also this one should extend EloquentRepositoryInterface.